### PR TITLE
Pyramid 1.6 & flake8 support

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR update quite a lot of eggs (notably Pyramid) and adds flake8 support to check that the Python code is pep8 compatible.

@voisardf, please review (and test!!! )

### CODE REVIEW

File: crdppf/__init__.py [hunk 1]
```diff
@@ -76,8 +81,8 @@ def includeme(config):
     config.add_route('getTownList', 'getTownList')
     config.add_route('getTopicsList', 'getTopicsList')
     config.add_route('createNewDocEntry', 'createNewDocEntry')
-    config.add_route('getDocumentReferences', 'getDocumentReferences')
-    config.add_route('getLegalDocuments', 'getLegalDocuments')
+    config.add_route('document_ref', 'getDocumentReferences')
+    config.add_route('legal_documents', 'getLegalDocuments')
```
Comment (voisardf) [inline crdppf/__init__.py]: Any particular reason you changed the names I should be aware of?

Comment (voisardf) [inline crdppf/__init__.py]: ok, got it, sorry!

Comment (kalbermattenm) [inline crdppf/__init__.py]: Yes... per flake8, functions do not contain any capital letters and `config.add_route` in fact references a function...

Basically, you should never use capital letters in your route names...

**Comment by kalbermattenm** [comment]
@voisardf, for your information, `crdppf/util/documents.py` has been checked by flake... (I did not do the rest of the files, as this would mean quite a lot of work...)

**Comment by kalbermattenm** [comment]
Fix #93

File: crdppf/__init__.py [hunk 2]
```diff
@@ -87,14 +92,9 @@ def includeme(config):
     config.add_route('ogcproxy', '/ogcproxy')
     
     # CLIENT VIEWS
-    config.add_view('crdppf.views.entry.Entry', route_name = 'home')
     config.add_view('crdppf.views.entry.Entry', route_name = 'images')
     config.add_view('crdppf.views.entry.Entry', route_name='test')
 
-    # ADMIN VIEWS
```
Comment (voisardf) [inline crdppf/__init__.py]: I don't get this removals yet? Again function vs. view?

**Comment by voisardf** [comment]
looks good to me otherways. thanks a lot!
I'll do some serious testing before we merge.

File: crdppf/__init__.py (hunk 2 above)
Comment (kalbermattenm) [inline crdppf/__init__.py]: No... if just over your function you define the `@view_config`, then you do not need to define `config.add_view`. These declaration do exactly the same... (they define a view...). This is redundant... and Pyramid 1.6 does not like it... (it crashes...)

For me, the `@view_config` is clearer, thus I would propose that we keep using it...

**Comment by kalbermattenm** [comment]
@voisard, it would good to know if this is OK to be merged...

Because the egg versions will soon be outdated again... and it would be good to install that on the new prod server... (and not use Pyramid 1.3 anymore...)

Just for info, the latest Pyramid version is 1.6 and will soon be 1.7...

**Comment by voisardf** [comment]
I flaked my branch and it looks good :)

**Comment by kalbermattenm** [comment]
So... Should I merge that ?

**Comment by kalbermattenm** [comment]
OK, confirmed, merging

**Pull request merged**